CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 9.7.3
+Throw TypeError on nulls, as NullThrownError is deprecated in Dart 2.19.
```

Comment:
Consider using `` `TypeError` `` and `` `NullThrownError` `` for source names.

Reply:
Ack, done

Reply:
If switching to `StateError`, remember to update this too.